.github/workflows/publish.yml

```
@@ -7,11 +7,20 @@ jobs:
     runs-on: ubuntu-latest
     name: "Publish a new version"
     if: github.event.label.name == 'accepted' && github.event.issue.state == 'open'
+    env:
+      SENTRY_DSN: "https://303a687befb64dc2b40ce4c96de507c5@o1.ingest.sentry.io/6183838"
     steps:
       - name: "Get repo contents"
         uses: actions/checkout@v2
         with:
           path: ".__publish__"
+
+      - name: "Install Sentry SDK"
+        - uses: actions/setup-node@v2
+          with:
+            node-version: 14
```

https://github.com/actions/setup-node#caching-packages-dependencies 😉 

```suggestion
            node-version: 14
            cache: npm
```